hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/WritableTokenStore.java

```diff
@@ -36,7 +36,7 @@
  * <p>This class is not exported from the module. It is an internal implementation detail.
  * This class is not complete, it will be extended with other methods like remove, update etc.,
  */
-public class WritableTokenStore {
+public class WritableTokenStore extends ReadableTokenStoreImpl {
```

Comment:
Since we extended , probably can move `get` and `size` to ReadableTokenStoreImpl and remove here

Reply:
`ReadableTokenStoreImpl` actually doesn't have a `size` method right now, but I'll move the other

Reply:
I mean `sizeOfState()` can be in `ReadableTokenStoreImpl`. Not a blocker though